OpenSim/Common/Test/testDataTable.cpp

```diff
@@ -217,8 +217,8 @@ int main() {
     ASSERT(tableCopy.getNumRows()    == table.getNumRows());
     for(unsigned r = 0; r < table.getNumRows(); ++r)
         for(unsigned c = 0; c < table.getNumColumns(); ++c)
-            ASSERT(tableCopy.getRowAtIndex(r).getElt(0, c) ==
-                       table.getRowAtIndex(r).getElt(0, c));
+            ASSERT(tableCopy.getRowAtIndex(r)[c] ==
+                       table.getRowAtIndex(r)[c]);
```

Comment:
This is a lot nicer @klshrinidhi. Thank you for digging into the issue and providing a fix.